### DISCUSSION TITLE
Numlock key control on the login screen and numlockx app

### DISCUSSION THREAD

**Discussion by ineuw** [linuxmint/discussions#73] [Ideas]
Standard users cannot access the numlock activation setting on the login options. Since **numlockx** is always present in my installations, could this be activated without a root user? Or, moved to a user accessible location?

**Comment by Lurux** [comment]
Numlockx doesn't seem to be installed on fresh 22.3 installations, would be useful to include it and enable related setting by default.

**Reply by AtomicRobotMan0101** [reply]
100%

I find this a curious omission.

**Comment by ineuw** [comment]
Using LMC 22.2 currently, numlockx was in my installation script, so I can't check. However LMC suffers from a number of illogical design features a user such as the placement of disparate and diffused control locations. To me, it seems that the LMC project had many different approaches by different designers over the past decade. I am using LMC since 2015, and I always wondered why not integrate numlockx into the OS. MB and OS control of the numlock still baffles me.

**Reply by AtomicRobotMan0101** [reply]
Remember it's a meal by a thousand chefs. It's a strength and a weakness. 

There is no ultimate plan or design committee with an entire department of 50 just to decide the right shade of silver for a phone case.... it's 6 people and a horde of fans for E.V.E.R.Y.T.H.I.N.G.

It's been on my to-do list for ages, but it's not super important. Always bigger fish to fry.

**Comment by claudiux** [comment]
If a laptop does not have a numeric keypad, it would be confusing for the user if the numeric keypad were enabled by default.
